src/slider/styles.scss

```diff
@@ -184,9 +199,12 @@
   appearance: none;
   position: absolute;
   block-size: 0;
-  inline-size: calc(100% + mixins.$thumb-size);
-  outline: 0;
+  // inline-size: calc(100% + mixins.$thumb-size);
+  // margin-inline-start: calc(mixins.$thumb-size / -2);
+  inline-size: calc(100% + 16px);
```

Comment:
Is this supposed to be hardcoded to 16px?

Reply:
Updated to use variable.